source/dubregistry/dbcontroller.d

```diff
@@ -9,16 +9,21 @@ import dub.semver;
 import std.array;
 import std.algorithm;
 import std.exception;
+
 //import std.string;
 import std.typecons : tuple;
 import std.uni;
 import vibe.vibe;
 
+/// Package cache used for searching in-memory
+DbPackage[] pkgCache;
 
-class DbController {
+class DbController
+{
```

Comment:
whole file still has automatic unrelated formatting changes, not reviewing this with mixed in changes

Reply:
So, how do I "undo" the formatting changes? Is it really that hard to review when dfmt has run?

Reply:
I can begin by changing { on new line to after function and see how things turn out

Reply:
(I'm manually going through the code and reformatting it to match how it previously was)